code/datums/voice_announcements.dm

```diff
@@ -0,0 +1,174 @@
+GLOBAL_LIST_EMPTY(voice_announce_list)
+
+/datum/voice_announce
+	var/id
+	var/client/client
+	var/is_ai = FALSE
+	var/open = FALSE
```

Comment:
I fail to see where this variable is used

Reply:
> I fail to see where this variable is used

line 38

Reply:
By used I meant read